airflow-core/src/airflow/ui/src/components/Graph/TaskLink.tsx

```diff
@@ -29,9 +29,25 @@ export const TaskLink = forwardRef<HTMLAnchorElement, Props>(({ id, isGroup, isM
   const { dagId = "", runId, taskId } = useParams();
   const [searchParams] = useSearchParams();
 
-  // We don't have a task group details page to link to
   if (isGroup) {
-    return <TaskName isGroup={true} isMapped={isMapped} {...rest} />;
+    const GroupTaskName = <TaskName isGroup={true} isMapped={isMapped} {...rest} />;
```

Comment:
Let's just pass TaskName below. No need to make it a const first.

Reply:
We can now merge the two return RouterLink and instead just pass relevant props and add a `const pathname` variable we can pass.